tests/testthat/test-as-data-frame.R

```diff
@@ -96,31 +110,12 @@ with_mock_crunch({
         expect_equal(names(new_ds_df),
                      aliases(variables(new_ds)))
         expect_equal(ncol(new_ds_df), 6)
-        expect_silent(
-            expect_equal(names(as.data.frame(new_ds_df)),
-                         c("birthyr", "location", "subvar2", "subvar1", 
-                           "subvar3", "textVar", "starttime", "subvar2", 
-                           "subvar1", "subvar3")))
-        
```

Comment:
Same as the comment above: it would be good to be able to keep these tests running even without an API.